playwright/_impl/_connection.py

```diff
@@ -48,8 +47,12 @@ async def inner_send(
             params = {}
         callback = self._connection._send_message_to_server(self._guid, method, params)
 
-        done, pending = await asyncio.wait(
-            {self._connection._transport.on_error_future, callback.future},
+        done, _ = await asyncio.wait(
```

Comment:
if self._error:
  e = self._error
  self._error = None
  throw e
should also do it then.

Reply:
Fixed